crates/task-impls/src/consensus.rs

```diff
@@ -488,6 +489,18 @@ where
             self.cur_view = new_view;
             self.current_proposal = None;
 
+            // Poll the future leader for lookahead
+            let lookahead_view = new_view + LOOK_AHEAD;
+            if !self.quorum_exchange.is_leader(lookahead_view) {
```

Comment:
Technically yes, but I don't think it will practically be a problem.  We'll know the leaders at least 1 epoch out, which should be more than enough time to contact them.

Reply:
Gotcha, so it sounds like `LOOK_AHEAD` needs to be less than the epoch length.